qemu/tests/cfg/virtio_fs_share_data_hugepage.cfg

```diff
@@ -65,11 +65,11 @@
         - with_cache:
             variants:
                 - auto:
-                    fs_binary_extra_options += ",cache=auto"
+                    fs_binary_extra_options += " --cache=auto"
                 - always:
-                    fs_binary_extra_options += ",cache=always"
+                    fs_binary_extra_options += " --cache=always"
                 - none:
-                    fs_binary_extra_options += ",cache=none"
+                    fs_binary_extra_options += " --cache=never"
```

Comment:
Hi @zhaojunyao7 
Do we consider updating the variant name to `never`?
I prefer to update it since it is more earlier to understand the case purpose.
Pls let me know your opinions. Thanks.